code/stable/dblpend/src/cpp/Calculations.hpp

```diff
@@ -15,11 +15,11 @@
 using std::vector;
 
 /** \brief Calculates dependent variables (rad)
-    \param m_1 mass of the first object (kg)
-    \param m_2 mass of the second object (kg)
-    \param L_2 length of the second rod (m)
-    \param L_1 length of the first rod (m)
-    \return dependent variables (rad)
+    \param m_1 mass of the first object: the mass of the first object (kg)
+    \param m_2 mass of the second object: the mass of the second object (kg)
+    \param L_2 length of the second rod: the length of the second rod (m)
+    \param L_1 length of the first rod: the length of the first rod (m)
+    \return dependent variables: column vector of displacement of rods with its derivatives (rad)
```

Comment:
And indeed `\return` as well, though in this case it looks like it is the chunk itself that is odd (too generic?)